### ISSUE TITLE
Failed to load the pomodoro package

### ISSUE THREAD

**Issue by tamagottschi** [yoshiori/pomodoro#5]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.190.0
**System**: Mac OS X 10.10.2
**Thrown From**: [pomodoro](https://github.com/yoshiori/pomodoro) package, v0.4.3

### Stack Trace

Failed to load the pomodoro package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/Users/jan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:5:199)
  at /Users/jan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:5:3
  at Object.<anonymous> (/Users/jan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:46:5)
  at Object.<anonymous> (/Users/jan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/jan/.atom/packages/pomodoro/lib/pomodoro.coffee:3:16)
  at Object.<anonymous> (/Users/jan/.atom/packages/pomodoro/lib/pomodoro.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app/src/package.js:665:34)
  at /Applications/Atom.app/Contents/Resources/app/src/package.js:174:28
  at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:152:15)
  at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app/src/package.js:166:12)
  at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:355:14)
  at PackageManager.module.exports.PackageManager.loadPackages (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:325:14)
  at Atom.module.exports.Atom.startEditorWindow (/Applications/Atom.app/Contents/Resources/app/src/atom.js:631:21)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-bootstrap.js:12:8)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-bootstrap.js:23:4)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at window.onload (file:///Applications/Atom.app/Contents/Resources/app/static/index.js:54:25)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "solarized-light-syntax"
    ],
    "projectHome": "/Users/jan/Implementierungen",
    "disabledPackages": [
      "refactor",
      "coffee-refactor"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.8
atom-jasmine, v0.5.0
atom-language-xquery, v0.2.0
atomatigit, v1.2.3
autocomplete-plus, v2.9.0
coffee-eval, v0.8.0
go-plus, v3.3.6
language-docker, v1.1.3
language-gherkin-german, v0.5.2
language-jade, v0.3.0
language-pegjs, v0.5.0
language-swift, v0.3.0
open-last-project, v0.4.1
pdf-view, v0.17.0
pomodoro, v0.4.3
pretty-json, v0.3.2
terminal-status, v1.6.4
todo-show, v0.8.0
xml-formatter, v0.8.1

# Dev
No dev packages
```

**Comment by qayshp** [comment]
+1
Only started happening for me after upgrading Atom to 0.198.0

**Comment by yoshiori** [comment]
fix 90a0a0898a3c41293e3f23a3b0f0cdf5acba436f
